### PR TITLE
No changes , trial commit

### PR DESCRIPTION
**Pull request by victyr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,138 +1,110 @@
...
-      <input
-        name="email"
-        id="email"
-        type="email"
-        placeholder="Enter your email address"
-        required
-      />
-      <input id="submit" type="submit" value="Get Started" class="btn" />
-    </form>
+<main id="main">
```
Comment (victyr) [inline index.html:1]: I think its better to write your code within html tags

File: index.html [hunk 2]
```diff
@@ -1,138 +1,110 @@
...
+      id="image"
+      src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+      alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+    />
+    <figcaption id="img-caption">
+      Dr. Norman Borlaug, third from the left, trains biologists in Mexico on
+      how to increase wheat yields - part of his life-long war on hunger.
+    </figcaption>
+  </figure>
+  <section id="tribute-info">
```
Comment (victyr) [inline index.html:15]: Nice using Section tag, i think its more professional to put it inside a body tag

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @victyr  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please, ensure that the Pull Request you opened utilises the "Review changes" feature from GitHub instead of just competing the code. This helps users to know the PR state(Approved or requested changes). You can this short [Video](https://youtu.be/lSnbOtw4izI) for more understanding.
- [ ] I would also suggest you use summarised content as a message to the user who receives the review. You can pick an idea from mine(the look and the logic but not the content).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by victyr** [comment]
Hello Kandy
I need some guidance on what to do, do i have to re comment and send
another pull request or I can just add more comments and send a PR?

File: index.html (hunk 1 above)
Comment (victyr) [inline index.html:1]: I think its professional to write your code within html tags

File: index.html [hunk 3]
```diff
@@ -1,138 +1,110 @@
...
+      If you have time, you should read more about this incredible human being
+      on his
+      <a
+        id="tribute-link"
+        href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+        target="_blank"
+        >Wikipedia entry</a
+      >.
+    </h3>
   </section>
```
Comment (victyr) [inline index.html:108]: Great job using the section tag, but its better to put it with the body tag

**Review by victyr — COMMENTED** [review]
Good job so far,
Your project is professional 👍
Good commit history👍
There are some issues that you still need to work but you are almost there!

**Review by Kandy-Peter — APPROVED** [review]
Hi @victyr  ,

# APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏

Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

##  Optional suggestion

- [ ] Please, ensure that you include an explanation, a "why", for each requested change.

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**